helmctlr/helm.go

```diff
@@ -33,11 +33,13 @@ type Controller struct {
 	Helm        helm.Interface // Helm for talking with helm
 	Namespace   string         // Default namespace to deploy into. If empty it will default to "default"
 	ReleaseName string         // Prefix for the helm release name. Will look like ReleaseName-CR_Name
+	Wait        bool           // Whether or not to wait for resources during Update and Install before marking a release successful
+	WaitTimeout int64          // time in seconds to wait for kubernetes resources to be created before marking a release successful
 	logger      *zap.SugaredLogger
 }
 
 // NewController will return a configured Helm Controller
-func NewController(chartDir, ns, rn, host string, logger *zap.SugaredLogger) *Controller {
+func NewController(chartDir, ns, rn, host string, wait bool, waitto int64, logger *zap.SugaredLogger) *Controller {
```

Comment:
If we add more fields to this we should probably move it to a config object. The amount of input here is getting really large.